client/src/components/ui/button.tsx

```diff
@@ -6,19 +6,23 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import { cn } from '@/lib/classnames';
 
 const buttonVariants = cva(
-  'inline-flex items-center justify-center text-base lg:text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-green-700 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans',
+  'inline-flex items-center justify-center text-base lg:text-sm ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-green-700 focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 font-sans',
   {
     variants: {
       variant: {
         // From the UI kit
-        default: 'bg-gray-50 hover:bg-gray-150 rounded-lg',
-        primary: 'bg-gray-700 hover:bg-gray-800 text-white',
+        default: 'bg-gray-650 hover:bg-gray-600 rounded-lg',
```

Comment:
I'm suggesting removing the ring offset so that we avoid the white shadow around the button when focussed:
```suggestion
        default: 'bg-gray-650 hover:bg-gray-600 rounded-lg focus-visible:ring-offset-0',
```